src/status_im/contexts/preview/quo/wallet/token_input.cljs

```diff
@@ -28,63 +28,57 @@
-                                                         (money/fiat->crypto input-amount
-                                                                             conversion-rate))
-                                                        (utils/cut-fiat-balance-to-two-decimals
-                                                         (money/crypto->fiat input-amount
-                                                                             conversion-rate)))]
-                                                  (number/remove-trailing-zeroes
-                                                   new-value))))))
-            converted-value               (if crypto?
-                                            (utils/prettify-balance currency
-                                                                    (money/crypto->fiat input-amount
-                                                                                        conversion-rate))
-                                            (utils/prettify-crypto-balance
-                                             (or (clj->js token) "")
-                                             (money/fiat->crypto input-amount conversion-rate)
-                                             conversion-rate))]
+      (let [{:keys [currency token-symbol crypto? error?]} @state
+            [input-state set-input-state]                  (rn/use-state controlled-input/init-state)
+            input-amount                                   (controlled-input/input-value input-state)
+            swap-between-fiat-and-crypto                   (fn []
+                                                             (if crypto?
+                                                               (set-input-state #(controlled-input/->fiat
+                                                                                  %
+                                                                                  conversion-rate))
+                                                               (set-input-state
+                                                                #(controlled-input/->crypto
+                                                                  %
+                                                                  conversion-rate))))
+            converted-value                                (if crypto?
+                                                             (utils/prettify-balance currency
+                                                                                     (money/crypto->fiat
+                                                                                      input-amount
+                                                                                      conversion-rate))
+                                                             (utils/prettify-crypto-balance
+                                                              (or (clj->js token-symbol) "")
```

Comment:
if `(clj->js token-symbol)` is able to return `null` is because `token-symbol` was already `nil`, right? is there any other possibility?

I think a clearer expression would be:
```clojure
(if token-symbol 
  (utils/prettify-crypto-balance (clj->js token-symbol))
  "")
```

wdyt?

Reply:
Thanks for the review!

As for this particular point, `prettify-crypto-balance` takes three arguments and even if `token-symbol` is nil, we can create a string with amount but without token information. So we can't just skip this function call if `token-symbol` is nil.

Reply:
Thanks for the clarification! Please ignore this comment